playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -205,7 +213,7 @@ public void prepare(@NonNull PKMediaConfig mediaConfig) {
 
 
         boolean shouldSwitchBetweenPlayers = shouldSwitchBetweenPlayers(source);
-        this.sourceConfig = new PKMediaSourceConfig(source, contentRequestAdapter);
+        this.sourceConfig = new PKMediaSourceConfig(source, contentRequestAdapter, cea608CaptionsEnabled);
```

Comment:
I think this is not scalable, as we get more options this might get bloated, please come with a better idea